gprofiler/profilers/node.py

```diff
@@ -186,6 +186,7 @@ def create_debugger_socket(nspid: int, ns_link_name: str) -> WebSocket:
     sock.settimeout(10)
     _validate_ns_node(sock, ns_link_name)
     _validate_pid(nspid, sock)
+    logger.info(f"Created debugger socket for pid: {nspid} (nodejs)")
```

Comment:
No need to add `(nodejs)` - we log the module name which owns the logger, so this will look like `INFO: gprofiler.profilers.node: Created debugger socket for pid...` anyway.